src/frontend/web_application/src/store/selectors/user.js

```diff
@@ -0,0 +1 @@
+export const UserSelector = state => state.user;
```

Comment:
I prefer keep `UpperCamelCase` format for class and simple `camelCase` for functions